samples/nrf9160/http_application_update/prj.conf

```diff
@@ -45,7 +45,6 @@ CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
 
 # DFU Target
 CONFIG_DFU_TARGET=y
-CONFIG_DFU_TARGET_MCUBOOT_SAVE_PROGRESS=y
 
 # Settings
 CONFIG_SETTINGS=y
```

Comment:
Also, remove settings then as it is not needed.

Reply:
good point. Also removed NVS